classes/file_search.php

```diff
@@ -335,16 +307,7 @@ public static function combine_shard_output(files $parent): void {
             fclose($output);
 
             // Create new pluginfile.
-            $fs = get_file_storage();
-            $fileinfo = [
-                'contextid' => \context_system::instance()->id,
-                'component' => 'tool_advancedreplace',
-                'filearea'  => 'files',
-                'itemid'    => $parent->get('id'),
-                'filepath'  => '/',
-                'filename'  => $parent->get_filename(),
-            ];
-            $fs->create_file_from_pathname($fileinfo, $outputpath);
+            $parent->save_pluginfile($output);
```

Comment:
The param in save_pluginfile needs to be updated to $outputpath.

Reply:
Thanks @bwalkerl updated with both those fixes